lib/glueby/contract/tx_builder.rb

```diff
@@ -1,14 +1,16 @@
 module Glueby
   module Contract
     module TxBuilder
+      include Glueby::Internal::Wallet::TapyrusCoreWalletAdapter::Util
+
       # collect outputs in results so that sum of them exceeds up to the specified amount.
       # @param [Array] results of listunspent
       # @param [Numeric] amount
       # @return [(Numeric, Array)] sum value of outputs and outputs
       # @raise [InsufficientFunds] if result of listunspent is not enough to pay the specified amount
       def collect_outputs(results, amount)
         results.inject([0, []]) do |sum, output|
-          new_sum = sum[0] + (output['amount'] * 100_000_000)
+          new_sum = sum[0] + tpc_to_tapyrus(output['amount'])
```

Comment:
The conversion process itself should be done in `Timestamp::Util#listunspent`.

Reply:
ok